e2e/resources/knative-service.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: v1
-data:
-  config: |
-    Host *
-      StrictHostKeyChecking no
 kind: ConfigMap
 metadata:
   name: m2k-ssh-config
+data:
```

Comment:
The ci complains for this:
```
Error from server (BadRequest): error when creating "e2e/resources/knative-service.yaml": ConfigMap in version "v1" cannot be handled as a ConfigMap: strict decoding error: unknown field "spec"
service.serving.knative.dev/m2k-save-transformation-func created
```

Reply:
which is pretty funny since there isn't a real change from the previous version (only location was changed)